internal/pod_models.go

```diff
@@ -225,6 +227,8 @@ func configureContainersForUnifiedImages(cluster *fdbv1beta2.FoundationDBCluster
 		"--copy-binary", "fdbcli",
 		"--log-path", "/var/log/fdb-trace-logs/monitor.log",
 	}
+	sidecarContainer.Env = append(sidecarContainer.Env, corev1.EnvVar{Name: "FDB_NETWORK_OPTION_TRACE_LOG_GROUP", Value: logGroup})
+	sidecarContainer.Env = append(sidecarContainer.Env, corev1.EnvVar{Name: "FDB_NETWORK_OPTION_TRACE_ENABLE", Value: "/var/log/fdb-trace-logs"})
```

Comment:
unclear if sidecare matters, I don't know of a case where someone would fdbcli the sidecar specifically myself

Reply:
Hm, yeah we can remove it from the sidecar.